### PR TITLE
added simple spend condition to TxLib

### PR DESCRIPTION
**Pull request by johannbarbie**
resolves https://github.com/leapdao/leap-contracts/issues/145

### CODE REVIEW

File: test/txLib.js [hunk 1]
```diff
@@ -11,12 +11,17 @@ import BN from 'bn.js';
 import { BigInt, equal } from 'jsbi';
 import { Period, Block, Tx, Input, Output, Outpoint, Type } from 'leap-core';
 import chai from 'chai';
+const ethers = require('ethers');
```
Comment (troggy) [inline test/txLib.js]: why ethers?

File: contracts/mocks/TxMock.sol [hunk 2]
```diff
@@ -19,25 +19,10 @@ contract TxMock {
   using TxLib for TxLib.Tx;
   using TxLib for TxLib.TxType;
 
-  function parse(bytes32[] memory _proof)
-    public pure returns (uint256 txType, bytes32[] memory rsp) {
+  function parse(bytes32[] memory _proof) public pure returns (TxLib.Tx memory txn) {
```
Comment (troggy) [inline contracts/mocks/TxMock.sol]: <3

Comment (troggy) [inline contracts/mocks/TxMock.sol]: ❤️

File: test/txLib.js (hunk 1 above)
Comment (johannbarbie) [reply test/txLib.js]: to be able to read structs from contract responses

**Pull request merged**